src/games/picture-match/Board/MatchingBoard.js

```diff
@@ -63,6 +63,12 @@ const MatchingBoard = () => {
           </Transition>
         </Grid.Column>
       </Grid.Row>
+      <div style={{ display: "none" }}>
```

Comment:
Do we need to add it in `JSX`, maybe adding this loop in `useEffect` will just work?

Reply:
Oh right. I had it in jsx, because I was adding images to DOM in the beginning. Will check

Reply:
Moved this to `useEffect`. Seems to work fine